janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto

```diff
@@ -21,6 +21,56 @@ import "google/protobuf/wrappers.proto";
 option java_multiple_files = true;
 option java_package = "org.janusgraph.graphdb.grpc.types";
 
+enum PropertyDataType {
+    PROPERTY_DATA_TYPE_UNSPECIFIED = 0;
+    // java class: String.class
+    PROPERTY_DATA_TYPE_STRING = 1;
+    // java class: Character.class
+    PROPERTY_DATA_TYPE_CHARACTER = 2;
+    // java class: Boolean.class
+    PROPERTY_DATA_TYPE_BOOLEAN = 3;
+    // java class: Byte.class
+    PROPERTY_DATA_TYPE_INT8 = 4;
+    // java class: Short.class
+    PROPERTY_DATA_TYPE_INT16 = 5;
+    // java class: Integer.class
+    PROPERTY_DATA_TYPE_INT32 = 6;
+    // java class: Long.class
+    PROPERTY_DATA_TYPE_INT64 = 7;
+    // java class: Float.class
+    PROPERTY_DATA_TYPE_FLOAT32 = 8;
+    // java class: Double.class
+    PROPERTY_DATA_TYPE_FLOAT64 = 9;
+    // java class: Date.class
+    PROPERTY_DATA_TYPE_DATE = 10;
+    // java class: Geoshape.class
+    PROPERTY_DATA_TYPE_GEO_SHAPE = 11;
+    // java class: UUID.class
+    PROPERTY_DATA_TYPE_UUID = 12;
+    // java class: Object.class
+    PROPERTY_DATA_TYPE_JAVA_OBJECT = 13;
+}
+
+message VertexProperty {
+    // The Internal id.
+    google.protobuf.Int64Value id = 1;
+    // The VertexProperty Name.
+    string name = 2;
+    // The Data type.
+    PropertyDataType data_type = 3;
+    enum Cardinality {
+        CARDINALITY_UNSPECIFIED = 0;
+        // Vertex can have the property just once.
+        CARDINALITY_SINGLE = 1;
+        // Vertex can have the multiple property of the same.
```

Comment:
```suggestion
        // Vertex can have multiple properties of the same name.
```

same below

Reply:
Your updated commit fixed this comment here, but the comment two lines below is exactly like this.